File: VB/SalaryReports - Step 11/UserControls/OptionsUserControl.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Linq
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Helpers

Namespace SalaryReports.UserControls
	Partial Public Class OptionsUserControl
		Inherits DevExpress.XtraEditors.XtraUserControl
		Public Sub New()
			InitializeComponent()
			SkinHelper.InitSkinGallery(galleryControl1)
		End Sub
	End Class
End Namespace

File: CS/SalaryReports/UserControls/OptionsUserControl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Linq;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Helpers;

namespace SalaryReports.UserControls
{
    public partial class OptionsUserControl : DevExpress.XtraEditors.XtraUserControl
    {
        public OptionsUserControl()
        {
            InitializeComponent();
            SkinHelper.InitSkinGallery(galleryControl1);
        }
    }
}
